app/src/components/BaseTabs.vue

```diff
@@ -0,0 +1,31 @@
+<!-- BaseTabs
+a navigation bar that can have various nav items with optional (INT) numbers,
+where one is the "active" by apply "border-b-2" class on it.
+-->
+
+<template>
+  <div class="px-4 md:px-12 py-6 border-b border-grey-100 flex flex-wrap gap-x-10 gap-y-2">
+    <span class="tab active">All (543)</span>
+    <span class="tab">Infra Tech (184)</span>
+    <span class="tab">dApp Tech (184)</span>
```

Comment:
What is the plan for un-hardcoding these numbers?

Reply:
I initially added this file as it was since I was showing this hardcoded "categories". But after weds call we decided to remove those categories filter for now. So I've just removed this file for now if that's ok.